packages/core/addon/models/visualization.ts

```diff
@@ -36,9 +36,11 @@ export default class VisualizationFragment extends Fragment {
     set(this, '_request', request);
     //TODO Add validation mixin
     //@ts-ignore
-    this.validateSync();
-    //@ts-ignore
-    return get(this, 'validations.isValid');
+    const { validations } = this.validateSync?.() || {};
+    if (validations) {
+      return validations.isValid;
+    }
+    return true;
```

Comment:
Just to be sure I'm understanding. we are returning true here because in this case there are no validations to run?

Reply:
Exactly, but I ended up adding back validations to metric label so we could revert to our assumption that there are validations to keep it simple